src/Cake.Issues.Reporting.Sarif.Tests/SarifIssueReportGeneratorTests.cs

```diff
@@ -874,28 +874,21 @@ public void Should_Generate_Report_With_BaseLineState_Unchanged_If_BaselineGuid_
 
                 };
 
-            fixture.SarifIssueReportFormatSettings.ExistingIssues.AddRange(existinIssues);
+            fixture.SarifIssueReportFormatSettings.ExistingIssues.AddRange(existingIssues);
 
             // When
             var logContents = fixture.CreateReport(issues);
 
             // Then
             var sarifLog = JsonConvert.DeserializeObject<SarifLog>(logContents);
 
-            // There are two runs because there are two issue providers.
-            sarifLog.Runs.Count.ShouldBe(1);
-
-            var run = sarifLog.Runs[0];
-            run.Tool.Driver.Name.ShouldBe("ProviderType Foo");
-            run.Results.Count.ShouldBe(1);
-
-            var result = run.Results[0];
-            result.Message.Text.ShouldBe("Message Foo.");
+            var run = sarifLog.Runs.ShouldHaveSingleItem();
+            var result = run.Results.ShouldHaveSingleItem();
```

Comment:
Make it sense to check the item value itself like on line 925?

Reply:
There are already tests in place which test if message is set correctly, no need to test it again here. On line 925 it is not primary testing the message, but reading the result (as there are two).